api/v1alpha1/contour_types.go

```diff
@@ -302,6 +317,75 @@ const (
 	GCPLoadBalancerProvider   LoadBalancerProviderType = "GCP"
 )
 
+type AwsLoadBalancerParameters struct {
+	// AllocationID is a list of Allocation IDs of Elastic IP addresses that are
+	// to be assigned to created Network Load Balancer.
+	// Example: "eipalloc-<xxxxxxxxxxxxxxxxx>"
+	//
+	// See: https://docs.aws.amazon.com/eks/latest/userguide/load-balancing.html
+	//
+	// +optional
+	AllocationID []string `json:"allocationId,omitempty"`
+}
+
+type AzureLoadBalancerParameters struct {
+	// Address is the desired IP of LoadBalancer. If LoadBalancerScope is Internal,
```

Comment:
s/Internal/"Internal"/

Reply:
You may also want to mention that the address must already exist, i.e. `az network public-ip create`.

Reply:
we want to ref the field name instead of the type in godocs: s/LoadBalancerScope/scope/ (repeat for other fields).

Reply:
Fixed